Makefile

```diff
@@ -135,7 +135,7 @@ BASE_IMAGES_TAG := $(shell awk '/base_images_tag/ \
 	{split($$NF, a, "\""); print a[2]}' base-images-tag.json)
 
 test-images: docker-images presto-server-rpm.rpm
-	python tests/product/image_builder.py $(IMAGE_NAMES)
+	PYTHONPATH=${PYTHONPATH}:$(shell pwd) python tests/product/image_builder.py $(IMAGE_NAMES)
```

Comment:
I got an error when I applied this patch locally:
ImportError: No module named docker
Makefile:138: recipe for target 'test-images' failed
make: *** [test-images] Error 1

It works both with and without the patch if I'm in my prestoadmin virtualenv